paddle/fluid/framework/downpour_worker.cc

```diff
@@ -689,7 +692,7 @@ void DownpourWorker::TrainFilesWithProfiler() {
             *thread_scope_, tid, features_[tid], feature_labels_[tid],
             sparse_key_names_[tid], sparse_grad_names_[tid], table.emb_dim(),
             &feature_grads_[tid], &push_sparse_status_, cur_batch, use_cvm_,
-            dump_slot_, &sparse_push_keys_[tid], no_cvm_);
+            dump_slot_, &sparse_push_keys_[tid], no_cvm_, place_, sparse_grad_region_);
```

Comment:
Could you put all the memory within a variable so that we can get the variable by name in a scope. If you add a sparse_grad_region_ here, you may have to add more in the future. please use names and scopes to get variables. btw, don't use the word region in this code, it is a concept in PSLIB which can not be easily understood in Paddle code.